easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb

```diff
@@ -101,13 +90,6 @@ exts_list = [
     }),
 ]
 
-# can't run tests in sanity check step, because build dependencies must be available
-postinstallcmds = [
-    "cd %%(builddir)s/jax-jaxlib-v%s && "
-    "PYTHONPATH=%%(installdir)s/lib/python%%(pyshortver)s/site-packages:$PYTHONPATH "
-    "pytest -n auto tests" % local_jaxlib_ver,
```

Comment:
We still need to run the tests through this easyconfig, can't be done in the `jaxlib` easyblock since `jax` is also required.

I think @Flamefire is looking into a better way of doing this, via `runtest` for `jax`?